connector/src/main/scala/com/basho/riak/spark/writer/WriteConf.scala

```diff
@@ -31,4 +37,9 @@ object WriteConf {
       writeQuorum = conf.getInt(WriteQuorumProperty, DefaultWriteQuorum)
     )
   }
+  
+  def fromOptions(options: Map[String, String], conf: SparkConf): WriteConf = {
```

Comment:
I get compilation exception
in object WriteConf, multiple overloaded alternatives of method apply define default arguments.

For 
```scala
case class WriteConf( writeQuorum: Int = WriteConf.DefaultWriteQuorum) 
```
Scala automatically creates apply(writeQuorum: Int = WriteConf.DefaultWriteQuorum) method. Hence the exception.

The same goes for ReadConf and this method is used in several places.